packages/commons/tests/unit/Utility.test.ts

```diff
@@ -81,10 +77,10 @@ describe('Class: Utility', () => {
   });
 
   describe('Method: isColdStart', () => {
```

Comment:
Is this block repeated?

Reply:
It's very similar to getColdStart(), and in the future we could remove one of the two to be honest, but for now we need to test both for coverage.